howto/install/customise.md

```diff
@@ -195,41 +64,40 @@ anbox-stream-gateway:
     snap_proxy: https://snap-proxy.example.com
```

Comment:
Identation is invalid in this YAML snippet

Reply:
Not sure what is going on here - https_proxy and snap_proxy have the same indentation in the file, so it must be a display issue.